options.go

```diff
@@ -193,7 +195,6 @@ type Options struct {
 	Compression    int
 	Zoom           int
 	Crop           bool
-	SmartCrop      bool
```

Comment:
The downside of not removing it is that this could drag on for a while. I can't think of a compromise solution as of yet though.
Two questions:

- Should I mark the SmartCrop flag as deprecated?
- What is the recommended approach for communicating deprecation?